pwa/src/components/organizationCard/OrganizationCard.module.css

```diff
@@ -0,0 +1,68 @@
+.container {
+  border: var(--web-app-component-card-border-color);
+  padding-inline-start: var(--web-app-size-md);
+  padding-inline-end: var(--web-app-size-md);
+  padding-block-start: var(--web-app-size-md);
+  padding-block-end: var(--web-app-size-md);
+}
+
+.container > *:not(:last-child) {
+  margin-block-end: var(--web-app-size-md);
+}
+
+.header {
+  display: flex;
+}
+
+.header > *:not(:last-child) {
+  margin-inline-end: var(--web-app-size-md);
+}
+
+.header > .headerContent {
+  flex: 1;
+}
+
+.titleLink {
+  font-size: var(--web-app-font-size-lg);
+  --denhaag-link-icon-gap: var(--web-app-size-xs);
+}
+
+.titleLink > :first-child > span > svg {
+  width: var(--web-app-font-size-lg);
+  height: var(--web-app-font-size-lg);
+}
+
+.titleLink > :first-child {
+  display: flex;
+}
+
+.description {
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.tagsContainer > *:not(:last-child) {
+  margin-inline-end: var(--web-app-size-xs);
+}
+
+.tagsContainer > div {
+  margin-block-end: var(--web-app-size-xs);
+}
+
+.logoContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: var(--web-app-organization-card-logo-height);
+  width: var(--web-app-organization-card-logo-width);
+}
+
+.logo {
+  vertical-align: middle;
+  max-width: 100%;
+  max-height: 100%;
```

Comment:
You actually _want_ these to fill their container, I think. So `width: ...` instead of `max-width` would be better here (same goes for height).